aws/resource_aws_glue_job.go

```diff
@@ -66,6 +66,10 @@ func resourceAwsGlueJob() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"glue_version": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
This new argument is currently causing a difference for existing configurations since its set to perform drift detection and Glue assigns a default value, e.g.

```
--- PASS: TestAccAWSGlueJob_GlueVersion (31.34s)
--- FAIL: TestAccAWSGlueJob_Command (31.54s)
--- FAIL: TestAccAWSGlueJob_DefaultArguments (45.49s)
--- FAIL: TestAccAWSGlueJob_MaxRetries (47.57s)
--- FAIL: TestAccAWSGlueJob_Description (54.14s)
--- FAIL: TestAccAWSGlueJob_SecurityConfiguration (56.34s)
--- FAIL: TestAccAWSGlueJob_ExecutionProperty (59.86s)
--- PASS: TestAccAWSGlueJob_Basic (65.38s)
--- FAIL: TestAccAWSGlueJob_PythonShell (67.25s)
--- FAIL: TestAccAWSGlueJob_MaxCapacity (74.10s)
--- FAIL: TestAccAWSGlueJob_Timeout (74.40s)
--- FAIL: TestAccAWSGlueJob_AllocatedCapacity (81.45s)
```

These failures look like:

```
--- FAIL: TestAccAWSGlueJob_Description (54.14s)
    testing.go:569: Step 1 error: After applying this step, the plan was not empty:
        
        DIFF:
        
        UPDATE: aws_glue_job.test
... omitted for clarity ...
          glue_version:                             "0.9" => ""
... omitted for clarity ...
```

While we could set `Default: "0.9"` on this attribute, I'm actually going to suggest `Computed: true` since I would imagine that over time, Glue will update the default version number assigned to Jobs that omit the parameter.

```suggestion
				Optional: true,
				Computed: true,
```

This should get the existing testing passing.